backend/models/form.py

```diff
@@ -8,6 +9,55 @@
 PUBLIC_FIELDS = ["id", "features", "questions", "name", "description"]
 
 
+class _WebHook(BaseModel):
+    """Schema model of discord webhooks."""
+    url: str
+    message: t.Optional[str]
+
+    @validator("url")
+    def validate_url(cls, url: str) -> str:
+        """Checks if discord webhook urls are valid."""
+        if not isinstance(url, str):
+            raise ValueError("Webhook URL must be a string.")
+
+        if "discord.com/api/webhooks/" not in url:
+            raise ValueError("URL must be a discord webhook.")
+
+        # Attempt to connect to URL
+        try:
+            httpx.get(url).raise_for_status()
+
+        except httpx.RequestError as e:
+            # Catch exceptions in request format
+            raise ValueError(
+                f"Encountered error while trying to connect to url: `{e}`"
+            )
```

Comment:
Not sure how I feel about synchronous validation here. This has the potential to block the worker from handling any requests.

If we can't use asyncio in the validator maybe we should write our own utility for validating and then call that before even creating the webhook model.

Reply:
Addressed in d1cb420, but I'll leave this open a little longer in case anyone has a better implementation.